### PR TITLE
Added some unit tests

### PR DESCRIPTION
**Pull request by T1duS**
Added unit tests for cli.py. If you think these are fine, I'll add for tab.py as well

### CODE REVIEW

**Comment by aadibajpai** [comment]
@pep8 resume

**Comment by aadibajpai** [comment]
@pep8speaks suggest diff

**Comment by T1duS** [comment]
@pep8speaks

**Comment by aadibajpai** [comment]
@pep8speaks suggest diff

**Comment by aadibajpai** [comment]
Looks ok, I'll test it once tomorrow 👍

File: swaglyrics/tests.py [hunk 1]
```diff
@@ -19,10 +27,49 @@ def test_that_stripping_works(self):
...
+
+		# Deleting above lyrics from unsupported.txt
+		f = open("unsupported.txt","r")
+		lines = f.readlines()
+		f.close()
+		f = open("yourfile.txt","w")
+		for line in lines:
+			if line not in ["Hello by World\n","Foo by Bar\n","Fantastic by Beasts\n"]:
+				f.write(line)
+        f.close()
```
Comment (aadibajpai) [inline swaglyrics/tests.py]: indentation looks off

File: swaglyrics/cli.py [hunk 2]
```diff
@@ -74,7 +74,9 @@ def lyrics(song, artist):
 	"""
 	if song and artist:  # check if song playing
 		try:
-			if song in open('unsupported.txt').read():
+			f = open('unsupported.txt')
```
Comment (aadibajpai) [inline swaglyrics/cli.py]: please revert these changes

File: swaglyrics/cli.py [hunk 3]
```diff
@@ -74,7 +74,9 @@ def lyrics(song, artist):
...
+			if song in f.read():
```
Comment (aadibajpai) [inline swaglyrics/cli.py]: and this

**Comment by aadibajpai** [comment]
Good job @T1duS! I'm waiting for those `tab.py` tests now 😛
